### PR TITLE
Fixes spraying/painting over dark paint

### PR DESCRIPTION
**Pull request by 81Denton**
## About The Pull Request
Spraycans/crayons had a very annoying bug where you're unable to paint over dark paint. I also added an infinite spraycan for admemes.

## Changelog
:cl: Denton
fix: Fixed a bug where players were unable to spray/paint over splotches of dark paint.
add: Added an infinite spraycan for admins to spawn.
/:cl:

### CODE REVIEW

**Comment by nemvar** [comment]
Infinite spraycan is a stealthy xenobio buff. https://github.com/tgstation/tgstation/pull/46519/files for reference.

**Comment by 81Denton** [comment]
Fucking xenobio I swear to g*d

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -666,8 +666,8 @@
 
 	if(isobj(target))
 		if(actually_paints)
-			if(color_hex2num(paint_color) < 350 && !istype(target, /obj/structure/window)) //Colors too dark are rejected
-				to_chat(usr, "<span class='warning'>A colour that dark on an object like this? Surely not...</span>")
+			if(color_hex2num(paint_color) < 350 && !istype(target, /obj/structure/window) && !istype(target, /obj/effect/decal/cleanable/crayon)) //Colors too dark are rejected
+				to_chat(usr, "<span class='warning'>A color that dark on an object like this? Surely not...</span>")
```
Comment (actioninja) [inline code/game/objects/items/crayons.dm]: based

Comment (81Denton) [reply code/game/objects/items/crayons.dm]: It's 50% for consistency and 50% for annoying British players :^)

**Pull request merged**